[TILE] Lightstep incident response TILE only integration with Datadog (#1243)
DataDog/integrations-extras: lightstep_incident_response/README.md

# Lightstep Incident Response

## Overview

> [Lightstep Incident Response][1] seamlessly orchestrates alert triaging and on-call scheduling across Slack, Zoom, desktop, and mobile. Provides unified response context with integrated collaboration alongside on-call, alert, and incident management workflows.

Reviewer: ```suggestion
[Lightstep Incident Response][1] orchestrates alert triaging and on-call scheduling across Slack, Zoom, desktop, and mobile. Lightstep also provides unified response context with integrated collaboration along with on-call, alert, and incident management workflows.
```

Lightstep Incident Response ingests data from Datadog through standard APIs and finds anomalous events before your business is impacted.

## Setup

### Installation

> To configure the integration and send data to Lightstep Incident Response, see the [Lightstep Incident Response-Datadog Integration][2]. 

Reviewer: ```suggestion
To configure the integration and send data to Lightstep Incident Response, see the [Lightstep Incident Response-Datadog Integration documentation][2]. 
```

If you do not have a Lightstep Incident Response account, [sign up][3] to start your free trial.

## Support

Contact [Datadog support][4] or chat with a live agent after signing up in [Lightstep Incident Response][1].

> 

Reviewer: ```suggestion
## Further Reading

Additional helpful documentation, links, and articles:

- [Datadog integration with Incident Response][2]

```
